server/clip_server/tensorrt-flow.yml

```diff
@@ -9,4 +9,5 @@ executors:
       metas:
         py_modules:
           - executors/clip_tensorrt.py
+      timeout_ready: 6000000
```

Comment:
Please double check what's the meaning of `6000000`? If I'm correct, the default value is 6 min. let's change it to 15 min (or need more time?).

Reply:
In bj gpu sometimes it needs 20-30min, the default value is 600,000 https://docs.jina.ai/fundamentals/flow/yaml-spec/#executors

Reply:
np